earth_enterprise/src/fusion/autoingest/sysman/AssetOperation.cpp

```diff
@@ -63,8 +63,38 @@ void RebuildVersion(const SharedString & ref, bool graphOps) {
   }
 }
 
-void HandleTaskProgress(const TaskProgressMsg & msg, bool graphOps) {
-  if (graphOps) {
+void CancelVersion(const SharedString & ref, MiscConfig::GraphOpsType graphOps) {
```

Comment:
Yeah, I always wonder whether passing a reference at all makes sense because SharedStrings are designed to be easy to copy. But hypothetically we could eventually change refs to something that doesn't copy easily, at which point references are more efficient. I figure const references aren't worse than anything else, so I just go with that by default.